mysql-test/suite/tokudb.bugs/t/alter_external_lock_assert.test

```diff
@@ -20,7 +17,5 @@ ALTER TABLE t1_tokudb_tokudb_smallvc_250 MODIFY c1 DATE;
 ALTER TABLE t1_tokudb_tokudb_smallvc_250 MODIFY c1 DATE;
 UNLOCK TABLES;
 COMMIT;
-disable_warnings;
-DROP TABLE IF EXISTS i,p,t1_tokudb_tokudb_smallvc_250;
-enable_warnings;
+DROP TABLE IF EXISTS i,t1_tokudb_tokudb_smallvc_250;
```

Comment:
The test looks like it could be coming from RQG, and, the actual test is highly likely to be not in DROP TABLE. And indeed temp and regular tables do not share the namespace. I'd suggest
```
DROP TEMPORARY TABLE t1_tokudb_tokudb_smallvc_250;
DROP TABLE t1_tokudb_tokudb_smallvc_250, etc;
```

Reply:
That makes sense and your suggestion should work.  I was not aware that temp tables go into their own namespace.  Seems like a bad idea all around to me, but than I am not that familiar with temp tables and their use cases.

Reply:
Yes, that did it.  Pushed the change, test passes locally.  I don't think it is necessary to spawn a new jenkins job for this one minor logical change.